### DISCUSSION TITLE
Stress calculation in the MPM-MIXED

### DISCUSSION THREAD

**Discussion by zhangfengeo** [Anura3D/Anura3D_OpenSource#53] [Q&A] [answered]
In Anura3D the internal forces by means of the Gauss point integration(σav) was calculated to mitigate the cell-crossing instability, the averaged stress σav represents the stress in one element, I want to ask whether this process  will affect the stress distribution in this element,that will make the stress of different material point in one element more consistent ?

**Comment by ayerrocolom** [comment, marked as answer]
Basically, all material points in one element will have the same stress.
